### DISCUSSION TITLE
Windows 10 launch problem

### DISCUSSION THREAD

**Discussion by Rorkh** [WilliamVenner/gmpublisher#9] [Help] [answered]
When I launch it I got a white blank screen. Windows 10. How to fix?

**Comment by Rorkh** [comment, marked as answer]
Lmao. You must write requirements on by capital letters and place it on top of README. Im dumb, sorry.

**Comment by WilliamVenner** [comment]
I'm amazed my program made you upgrade to Windows 10 :D
